packages/ts/src/components/crosshair/index.ts

```diff
@@ -34,6 +34,9 @@ export class Crosshair<Datum> extends XYComponentCore<Datum, CrosshairConfigInte
   datumIndex: number
   show = false
   private _animFrameId: number = null
+  private _currentXData: number | Date | undefined = undefined
```

Comment:
When I read `this._currentXData` I expect it to have type `Datum`. Maybe `this._dataX` since we already have `this.x`?